File: src/bc-shipping/scheduler/src/com/fabrikam/dronedelivery/deliveryscheduler/scheduler/models/invoker/PackageDetail.java
package com.fabrikam.dronedelivery.deliveryscheduler.scheduler.models.invoker;

public class PackageDetail {
	private String Id;
	private PackageSize Size;
	
	public PackageDetail() {

	}

	public String getId() {
		return Id;
	}

	public void setId(String id) {
		Id = id;
	}

	public PackageSize getSize() {
		return Size;
	}

	public void setSize(PackageSize size) {
		Size = size;
	}
	
	public PackageDetail(String id, PackageSize size){
		this.Id = id;
		this.Size = size;
	}
}

File: src/shipping/dronescheduler/Fabrikam.DroneDelivery.DroneSchedulerService/Models/PackageDetail.cs
// ------------------------------------------------------------
//  Copyright (c) Microsoft Corporation.  All rights reserved.
//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
// ------------------------------------------------------------

using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Fabrikam.DroneDelivery.DroneSchedulerService.Models
{
    public class PackageDetail
    {
        public string Id { get; set; }
        public PackageSize Size { get; set; }
    }
}
